lib/chef/knife/oci_server_create.rb

```diff
@@ -212,7 +237,44 @@ def wait_for_ssh(hostname, ssh_port, interval_seconds, max_time_seconds)
         false
       end
 
-      def can_ssh(hostname, ssh_port)
+      # returns a Net::SSH:Gateway object or nil
+      def get_ssh_gateway(ssh_gateway)
+        # Probably should be doing a more complete lookup here.
```

Comment:
Seems fine for now.... do we know what the normal "knife bootstrap" behavior is for this?

Reply:
At least some of the other knife plugins will look into the user's .ssh/* configuration files (via the Net::SSH::Config methods) and can do more complicated things.  I think this is sufficient for now.  We can always extend this routine in the future if more complicated scenarios require it.